keps/sig-scheduling/4671-gang-scheduling/README.md

```diff
@@ -311,78 +340,116 @@ to identify pods belonging to it. However, with this pattern:
 
 The `Workload` type will be defined with the following structure:
 ```go
+// Workload allows for expressing scheduling constraints that should be used
+// when managing lifecycle of workloads from scheduling perspective,
+// including scheduling, preemption, eviction and other phases.
 type Workload struct {
 	metav1.TypeMeta
+	// Standard object's metadata.
+	// Name must be a DNS subdomain.
+	//
+	// +optional
 	metav1.ObjectMeta
+
+	// Spec defines the desired behavior of a Workload.
+	//
+	// +required
 	Spec WorkloadSpec
-	Status WorkloadStatus
 }
 
-// WorkloadSpec describes a workload in a portable way that scheduler and related
-// tools can understand.  
+// WorkloadMaxPodGroups is the maximum number of pod groups per Workload.
```

Comment:
This is the number of items in the list `PodGroups`

Reply:
FWIW - these updates are based on what was merged in the code.
I'm happy for these to be updated, but for these specifically maybe we should open a dedicated PR in k/k.